### PR TITLE
[DOCS] Note ESS must use custom bundles for custom GeoIP database files

### PR DESCRIPTION
**Pull request by predogma**
This is missing cloud context for custom databases.  Need to provide note and link to the https://www.elastic.co/guide/en/cloud/current/ec-custom-bundles.html which has example specifically for GeoIP bundle and adding custom database.

### CODE REVIEW

**Review by jrodewig — APPROVED** [review]
Thanks @predogma. I relocated your sentence so it's alongside the directory location for self-managed deployments.

Otherwise, LGTM. I'll wait for an engineer to approve before merging in.

**Review by martijnvg — APPROVED** [review]
LGTM

**Comment by jrodewig** [comment]
Thanks @martijnvg.

**Pull request merged**
